### ISSUE TITLE
Feature request: Option to strip process number argument

### ISSUE THREAD

**Issue by fadookie** [sveinbjornt/Platypus#55]
In Platypus 4.9 on Mac OS 10.10.3, when I create a droplet and run it, the first argument my script is getting is now something like `-psn_0_6292992` (the number varies.) This appears to be a Carbon thing indicating the process ID. I don't really care about this and would like to keep the complexity of my droplets low so they can simply iterate over the file arguments and process them.

For now, I found a workaround (assuming this argument is always present when the final app runs on any version of Mac OS, which I'm not certain of.) For shell scripts at least, add a line at the top of the script that says:

    shift

**Comment by fadookie** [comment]
This is odd, now I can't reproduce the issue.

**Comment by fadookie** [comment]
Now it's happening again. Possibly related to opening a saved profile and building the app bundle again. The profile happens to be outside the normal Profiles directory as I keep my Platypus scripts in separate git repos.

I whipped up a workaround in bash. It's verbose but it ignores the -p flag if it's present, and then the rest of the arguments can be used normally:

```bash
#!/bin/bash

# Parse out unwanted -p flag that may get sent by Platypus/Carbon
while getopts "p::" Option
do
  case $Option in
    p     ) echo "Scenario #4: option -p-\
                  with argument \"$OPTARG\"   [OPTIND=${OPTIND}]";;
    #  Note that option 'p' must have an associated argument,
    #+ otherwise it falls through to the default.
    *     ) echo "Unimplemented option chosen.";;   # Default.
  esac
done

#  Decrements the argument pointer so it points to next argument.
#  $1 now references the first non-option item supplied on the command-line
#+ if one exists.
shift $(($OPTIND - 1))

echo "File arguments: $@"
```

**Comment by sveinbjornt** [comment]
This is fixed in master and will be released as part of 5.0
